### PR TITLE
Details Tab in Meeting csharp

### PR DESCRIPTION
**Pull request by Trinetra-MSFT**
## Description

This sample app demonstrate the behavior of Details tab in meeting.

Language Used: C#

### Proposed Changes
1. Added a tab in meeting which basically Adds agenda items to the tab
2. User can add new agenda item
3. it will be added to the tab
4. user can send agenda item in chat
5. Participants in meeting can submit their response to agenda
6. they can also view the collected response in chat or Tab

##Testing
image

Adaptive Card in Chat
image

Response Recorded
image

### CODE REVIEW

File: samples/meetings-details-tab/csharp/DetailsTab/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            let taskInfo = {
+                title: null,
+                height: null,
+                width: null,
+                url: null,
+                card: null,
+                fallbackUrl: null,
+                completionBotId: null,
+            };
+            taskInfo.url = "https://5a584af2d305.ngrok.io/Detail";
```
Comment (Wajeed-msft) [inline samples/meetings-details-tab/csharp/DetailsTab/Views/Home/Index.cshtml:95]: Could you please fetch this from config file?

File: samples/meetings-details-tab/csharp/DetailsTab/Views/Shared/_EmbedPage.cshtml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width" />
```
Comment (Wajeed-msft) [inline samples/meetings-details-tab/csharp/DetailsTab/Views/Shared/_EmbedPage.cshtml:4]: Could you please remove this page if it's not used?

File: samples/graph-appcatalog-lifecycle/nodejs/README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+﻿### AppCatalog sample
+
+Bot Framework v4 AppCatalog sample.
```
Comment (Wajeed-msft) [inline samples/graph-appcatalog-lifecycle/nodejs/README.md:3]: This looks like additional readme file for App Catalog, could you please remove it?

File: samples/meetings-details-tab/csharp/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+title: Microsoft Teams C# Helloworld Sample
+description: Microsoft Teams "Hello world" application for .NET/C#
+extensions:
+  contentType: samples
+  platforms:
+  - CSS
+  createdDate: 10/16/2017 10:02:21 PM
+---
+
+# Official documentation
```
Comment (Wajeed-msft) [inline samples/meetings-details-tab/csharp/README.md:16]: Could you please add detailed documentation on setting up the samples locally?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Review by asnow003 — APPROVED** [review]

**Pull request merged**